packages/msgraph/test/Adaptor.test.js

```diff
@@ -452,3 +458,42 @@ describe('getFile', () => {
     });
   });
 });
+
+describe('uploadFile', () => {
+  it.skip('should convert array of object to excel and post to specified path', async () => {
```

Comment:
Is this supposed to be skipped?

Reply:
Yes, Since i need to cut a release soon, i have open an issue for it https://github.com/OpenFn/adaptors/issues/397